### PR TITLE
GG-150 Make active game design prettier

### PR DESCRIPTION
**Pull request by whole01**
* dartboard alignment on small screen adjusted
* fixed thrown points display
* Lined up buttons 
* changed headline

Setup page:
* Added margins between player-add components

* Translated some text to german

### CODE REVIEW

File: src/app/components/pages/game-active/game-active.component.html [hunk 1]
```diff
@@ -1,36 +1,35 @@
 <div class="container">
-  <h1>Placeholder!</h1>
+  <h1>Du bist an der Reihe: {{currentPlayerName}}!</h1>
 </div>
 
+<div class="container">
+
```
Comment (mohjohfox) [inline src/app/components/pages/game-active/game-active.component.html:6]: For what is this empty div container?

File: src/app/components/pages/game-active/game-active.component.html [hunk 2]
```diff
@@ -1,36 +1,35 @@
...
+</div>
 <div class="container">
   <app-leaderboard></app-leaderboard>
   <app-dartboard></app-dartboard>
+
 </div>
 
-<div class="container">
+<div class="playerScore container">
```
Comment (mohjohfox) [inline src/app/components/pages/game-active/game-active.component.html:15]: Don't do empty spaces in class names

Comment (whole01) [reply src/app/components/pages/game-active/game-active.component.html:15]: For a html component to have multiple classes you seperate the names of the classes with spaces. For reverence: https://www.w3schools.com/html/html_classes.asp

File: src/app/components/pages/game-active/game-active.component.html (hunk 1 above)
Comment (whole01) [reply src/app/components/pages/game-active/game-active.component.html:6]: Can be removed.

File: src/app/components/pages/game-active/game-active.component.html (hunk 2 above)
Comment (mohjohfox) [reply src/app/components/pages/game-active/game-active.component.html:15]: Oh sorry. Thanks for the hint :)

File: src/app/components/pages/winner/winner.component.html [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<h1>Winner: {{winner}}</h1>
+<h1>Gewonnen hat: {{winner}}</h1>
 <div class="container">
 <app-leaderboard></app-leaderboard>
```
[the remark this answers is not in the document]
Comment (whole01) [reply src/app/components/pages/winner/winner.component.html:5]: Is now aligned with div.

**Pull request merged**